### PR TITLE
Alternative Support Widget Experiment

### PR DESCRIPTION
**Pull request by AllTerrainDeveloper**
This is an alternative support widget experiment for demo purposes and accessible to Automatticians only.

It needs D113721-code applied in your sandbox in order to work.

### CODE REVIEW

**Review by gravityrail — APPROVED** [review]
This is a great first pass. Looking forward to it evolving quickly.

File: client/odysseus/style.scss [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+		transform: rotate(-2.4deg);
+	}
+	75% {
+		transform: rotate(1.2deg);
+	}
+}
+
+.environment-badge {
+	display: none !important;
+}
```
Comment (AllTerrainDeveloper) [inline client/odysseus/style.scss]: Oops

Comment (gravityrail) [reply client/odysseus/style.scss]: Fixed

File: client/my-sites/email/email-management/home/email-list-inactive.tsx [hunk 2]
```diff
@@ -31,6 +32,15 @@ const EmailListInactiveItem = ( { domain, source }: EmailListInactiveItemProps )
 					currentRoute,
 					source
 				) }
+				onClick={ () => {
+					sendNudge( {
+						nudge: 'email-comparison',
+						initialMessage: `I see you want to an email provider to your domain ${ domain.name }. I can give you a few tips on how to do that.`,
```
Comment (nb) [inline client/my-sites/email/email-management/home/email-list-inactive.tsx:38]: We are missing a verb here, I assume this needs to be “want to add an email provider”?

Comment (AllTerrainDeveloper) [reply client/my-sites/email/email-management/home/email-list-inactive.tsx:38]: Thank you! I will fix it with the next changes I commit :)

Comment (nb) [reply client/my-sites/email/email-management/home/email-list-inactive.tsx:38]: Thanks, @AllTerrainDeveloper.

**Pull request merged**